internal/pkg/crawl/crawl.go

```diff
@@ -163,11 +163,13 @@ func (c *Crawl) Start() (err error) {
 	} else {
 		// Push the seed list to the queue
 		c.Log.Info("Pushing seeds in the local queue..")
+		var seedPointers []*queue.Item
 		for _, item := range c.SeedList {
-			item := item
-			c.Queue.Enqueue(&item)
+			seedPointers = append(seedPointers, &item)
 		}
+		c.Queue.BatchEnqueueUntilCommited(seedPointers...) // TODO: chunk
 		c.SeedList = nil
+		seedPointers = nil
```

Comment:
nulling a pointer used in a function without waiting for the function return can be deadly, this is not safe

Reply:
`seedPointers` is just used to convert c.SeedList (`[]queue.Item`) to `[]*queue.Item` .
Anyway, `seedPointers = nil` is useless here, I deleted the line.